tests/lib/hw_unique_key_tfm/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if (CONFIG_HUK_TEST_BOARD_SNR STREQUAL "")
+	message(WARNING "If it is to be run on target, this test requires setting"
+			"CONFIG_HUK_TEST_BOARD_SNR")
+endif()
```

Comment:
This should be moved to come after 
```
find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
```
in which case you could combine it with 
```
if (NOT (CONFIG_HUK_TEST_BOARD_SNR STREQUAL ""))
add_custom_command(TARGET app
		POST_BUILD
     .....
```

like this:
```
if (CONFIG_HUK_TEST_BOARD_SNR STREQUAL "")
	message(WARNING "If it is to be run on target, this test requires setting"
			"CONFIG_HUK_TEST_BOARD_SNR")
else()
  add_custom_command(TARGET app
		POST_BUILD
		COMMAND
    .....
endif()